.github/CODEOWNERS

```diff
@@ -42,3 +42,6 @@ troubleshooting/kubernetes                               @Cray-HPE/platform-engi
 
 # Network Items
 /operations/network                                      @Cray-HPE/management-network
+
+# SAT Items
+/operations/sat                                          @Cray-HPE/system-admin-toolkit-admin
```

Comment:
I've now added this team, so this error will go away (maybe after merging).

Reply:
What do you mean by added this team?